docs/web/user_guide.md

```diff
@@ -194,6 +195,17 @@ optional arguments:
                         removing "/a/b/" prefix will store files like c/x.cpp
                         and c/y.cpp. If multiple prefix is given, the longest
                         match will be removed.
+  --config CONFIG_FILE  Allow the configuration from an explicit JSON based
+                        configuration file. The values configured in the
+                        config file will overwrite the values set in the
+                        command line. The format of configuration file is:
+                        {
+                          "store": [
+                            "--name=run_name",
+                            "--tag=my_tag"
```

Comment:
```suggestion
                            "--tag=my_tag",
```
A comma is missing, the json is invalid without it.

Reply:
Good catch, thanks!